### PR TITLE
store login method

### LINKED ISSUE #297: [frontend][web] Remember the login method used by the user
After the user logs in using method X (e.g. Facebook), a cookie must be set:
- Cookie name: loginmethod
- Value: "facebook" | "google" | "sms" (all lower case)

This cookie must be readable by JS, and as such, it must not have the flag "httponly" set.

### PR DESCRIPTION
**Pull request by diogoteix**
Closes #297

### CODE REVIEW

File: Trackovid19-web/src/app/screens/login/login.component.html [hunk 1]
```diff
@@ -14,6 +14,7 @@
       <div class="action" fxLayout="row">
         <div class="action-content">
           <div
+            *ngIf="!loginMethod || (loginMethod && loginMethod === 'facebook')"
```
Comment (nunofgm) [inline Trackovid19-web/src/app/screens/login/login.component.html]:
```suggestion
            *ngIf="!loginMethod || loginMethod === 'facebook'"
```

File: Trackovid19-web/src/app/helpers/local-storage.ts [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+  getShareStatus(): string {
+    return localStorage.getItem('share');
+  }
+
+  setGDPR(gdpr: string) {
+    localStorage.setItem('gdpr', gdpr);
+  }
+
+  getGPRD(): string {
```
Comment (nunofgm) [inline Trackovid19-web/src/app/helpers/local-storage.ts]: `setGDPR` and `getGPRD` have different acronyms

**Review by nunofgm — APPROVED** [review]

File: Trackovid19-web/src/app/screens/login/login.component.html (hunk 1 above)
Comment (diogoteix) [reply Trackovid19-web/src/app/screens/login/login.component.html]: Done!

File: Trackovid19-web/src/app/helpers/local-storage.ts (hunk 2 above)
Comment (diogoteix) [reply Trackovid19-web/src/app/helpers/local-storage.ts]: Done!

**Review by fabioscsilva — APPROVED** [review]

**Pull request merged**